userspace/libsinsp/container.cpp

```diff
@@ -1293,6 +1359,67 @@ string sinsp_container_manager::get_container_name(sinsp_threadinfo* tinfo)
 	return res;
 }
 
+void sinsp_container_manager::identify_healthcheck(sinsp_threadinfo *tinfo)
```

Comment:
Do I get it right? This function should return true/false in tinfo->m_is_container_healthcheck?
If so, it might be worth writing it as a comment.
Also lazily initializing the variable at the very beginning to false might be good practice (even though it seems to be defaulted to false in the constructor). Or we can just write it in the comment and only set it to true, removing the set to false.

Reply:
Yep, did the lazy init as you suggested, and added some comments.